Updating README.md (#383)
JuliaParallel/Dagger.jl: README.md

c = Dagger.@spawn sum(b)
fetch(c) # some number!
```
## Contributing Guide
[![PRs Welcome](https://img.shields.io/badge/PRs-welcome-brightgreen.svg?style=flat-square)](http://makeapullrequest.com)

> ![GitHub issues](https://img.shields.io/github/issues/JuliaParallel/Dagger.jl)

Reviewer: ```suggestion
[![GitHub issues](https://img.shields.io/github/issues/JuliaParallel/Dagger.jl)](https://github.com/JuliaParallel/Dagger.jl/issues)
```

> ![GitHub contributors](https://img.shields.io/github/contributors/JuliaParallel/Dagger.jl)

Reviewer: ```suggestion
[![GitHub contributors](https://img.shields.io/github/contributors/JuliaParallel/Dagger.jl)](https://github.com/JuliaParallel/Dagger.jl/graphs/contributors)
```

> [![GitHub](https://badgen.net/badge/icon/github?icon=github&label)](https://github.com)

Reviewer: ```suggestion
```
I don't think we need a GitHub badge, unless you have a particular reason you think it should be here?

Contributions are encouraged. 

There are several ways to contribute to our project:

**Reporting Bugs**: If you find a bug, please open an issue and describe the problem. Make sure to include steps to reproduce the issue and any error messages you receive regarding that issue.

**Fixing Bugs**: If you'd like to fix a bug, please create a pull request with your changes. Make sure to include a description of the problem and how your changes will address it.

Additional examples and documentation improvements are also very welcome.

## Documentation

> List of recommended Dagger resources:

Reviewer: Would you mind also listing the video links that I sent to you all in Slack? I think they're useful resources to point contributors to.

Author: yes, sure. I'll add them.

> - https://docs.juliahub.com/Dagger/0a2f8/0.13.3/

Reviewer: ```suggestion
- Docs [![][docs-master-img]][docs-master-url]
```

Since otherwise we're pointing to a specific, old commit's docs.

Author: done!

- https://juliaparallel.org/Dagger.jl/dev/

## Help and Discussion

> For help and discussion, we suggest asking on the following:

Reviewer: ```suggestion
For help and discussion, we suggest asking in the following places:
```

> [Julia Discourse](https://discourse.julialang.org/t/dagger-use-cases/72012) and on the [Julia Slack](https://julialang.org/slack/)

Reviewer: ```suggestion
[Julia Discourse](https://discourse.julialang.org/c/domain/parallel/34) and on the [Julia Slack](https://julialang.org/slack/) in the `#distributed` channel.
```

I changed the Discourse link to the `parallel` category, since it's more general, and that particular post you referenced isn't the best place for users to post questions or foster discussions.
